modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -8,60 +8,185 @@
 
 Query is a service that retrieves traces from storage and hosts the user interface to display them.
 
-.Parameters used by the {TempoOperator} to define Query
+.Configuration parameters for the Querier
 [options="header"]
-[cols="l, a, a, a"]
 |===
 |Parameter |Description |Values |Default value
 
-|spec:
-  query:
-    replicas:
-|Specifies the number of Query replicas to create.
-|Positive integer
+|`nodeSelector`
+|NodeSelector is the simplest recommended form of node selection constraint.
+|type: object, additionalProperties: type: string
 |
+
+|`replicas`
+| Replicas represents the number of replicas to create for this component.
+| type: integer, format: int32
+|
+
+|`tolerations`
+|Tolerations defines component specific pod tolerations.
+|type: array, x-kubernetes-list-type: atomic
+|
+
+|`Tolerations` (items)
+|The pod to which the tolerations are attached to tolerates any taint that matches the triple <key, value, effect> using the matching operator <operator>.
+|type: object
+|
+
+|`effect`
+| Effect indicates the taint effect to match for the toleration. Empty means match all taint effects. When specified, allowed values are NoSchedule, PreferNoSchedule, and NoExecute.
+|type: string
+|
+
+|`key`
+|Key is the taint key that the toleration applies to. Empty means match all taint keys. If the key is empty, operator must be Exists; this combination means to match all values and all keys.
```

Comment:
```suggestion
|Key is the taint key that the toleration applies to. Empty means match all taint keys. If the key is empty, the value for `operator` must be `Exists`; this combination means to match all values and all keys.
```

Reply:
/me confused, I thought we remove the `key` field (and all other properties of the `toleration`)?
(it's removed in the next suggestion)

Reply:
It's just an older suggestion.